### PR TITLE
need to allow CORS using config

### PR DESCRIPTION
**Pull request by monsieurleberre**

### CODE REVIEW

File: src/Trakx.MarketData.Server/Trakx.MarketData.Server.csproj [hunk 1]
```diff
@@ -74,4 +74,12 @@
     </None>
 
   </ItemGroup>
+
+  <ItemGroup>
```
Comment (AymericNoel) [inline src/Trakx.MarketData.Server/Trakx.MarketData.Server.csproj:78]: These line shouldn't be removed ? It's auto generation no?

File: src/Trakx.MarketData.Server/appsettings.json [hunk 2]
```diff
@@ -12,8 +12,9 @@
     }
   },
   "AllowedHosts": "*",
-  "IndexValuationUpdaterConfiguration": {
-    "PollingIntervalInSeconds": 3600
+  "MarketDataServerConfiguration": {
```
Comment (AymericNoel) [inline src/Trakx.MarketData.Server/appsettings.json:15]: Great !

**Review by AymericNoel — DISMISSED** [review]
Nice !

File: src/Trakx.MarketData.Server/Trakx.MarketData.Server.csproj (hunk 1 above)
Comment (monsieurleberre) [reply src/Trakx.MarketData.Server/Trakx.MarketData.Server.csproj:78]: oh that is an empty folder that I should remove to make this disappear

**Pull request merged**